rpc/grpc_example/main.go

```diff
@@ -85,16 +85,45 @@ func main() {
 	}
 
 	wsClient := pb.NewWalletServiceClient(conn)
-	validateAddrRequest := &pb.ValidateAddressRequest{
-		Address: "TcpEWwGdCN3RCNAQUhBn8f2Xdko2JzcQSQs", // ValidateAddress only sets ScriptType for owned and P2SH addresses.
+
+	for _, script := range importScripts {
+		// NOTE: These scripts will forever be imported into your
+		// testing wallet.
+		scriptB, err := hex.DecodeString(script)
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+		importScriptRequest := &pb.ImportScriptRequest{
+			Script: scriptB,
+		}
+		_, err = wsClient.ImportScript(context.Background(), importScriptRequest)
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
 	}
-	validateAddrResp, err := wsClient.ValidateAddress(context.Background(), validateAddrRequest)
+
+	// Add an owned address to validated address.
+	nextAddressResp, err := wsClient.NextAddress(context.Background(), new(pb.NextAddressRequest))
 	if err != nil {
 		fmt.Println(err)
 		return
 	}
-	fmt.Println(validateAddrResp.ScriptType) // unset / non-standard unless wallet-owned AND P2SH!
-	fmt.Println(prototext.MarshalOptions{Multiline: true}.Format(validateAddrResp))
+	validateAddrs = append(validateAddrs, nextAddressResp.Address)
+
+	for _, addr := range validateAddrs {
+		validateAddrRequest := &pb.ValidateAddressRequest{
+			Address: addr, // ValidateAddress only sets ScriptType for owned and P2SH addresses.
+		}
+		validateAddrResp, err := wsClient.ValidateAddress(context.Background(), validateAddrRequest)
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+		fmt.Println(validateAddrResp.ScriptType) // unset / non-standard unless wallet-owned AND P2SH!
```

Comment:
These two comments don't apply any more right?  You fixed this issue if I understand.

Reply:
Yes, will remove comments.

Reply:
Removed.